pages/create/index.js

```diff
@@ -1,20 +1,21 @@
 import styled from "styled-components";
 import { useState } from "react";
-import CreateFlowForm from "../../components/CreateFlow";
+import CreateFlowForm from "../../components/CreateFlowForm";
 import { nanoid } from "nanoid";
 import FlowCard from "../../components/FlowCard";
 import useLocalStorage from "../../hooks/useLocalStorage";
 import { flowDummys } from "../../db";
 
 export default function CreateFlow() {
-  const [open, setOpen] = useState(false);
+  const [openForm, setOpenForm] = useState(false);
+  const [editFormId, setEditFormId] = useState(null);
   const [flows, setFlows] = useLocalStorage("flows", flowDummys);
 
-  function toggleOpen() {
-    setOpen((prev) => (prev = !prev));
+  function toggleOpenForm() {
+    setOpenForm((prev) => (prev = !prev));
```

Comment:
```suggestion
    setOpenForm((prev) => !prev);
```
You're rewriting the parameter `prev`. You don't need to do that. All you need is to toggle the prev value.